src/bridges/mean/MeanBridge.sol

```diff
@@ -95,6 +96,53 @@ contract MeanBridge is BridgeBase, Ownable2Step {
         return (0, 0, true);
     }
 
+    /**
+     * @notice A function which will allow the users to close their DCA positions on Mean Finance
```

Comment:
Maybe we should add for `@dev` that position users will only be able to close position IF protocol is paused or position has ended.